src/components/Trade/Components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -0,0 +1,50 @@
+import { Collapse, Flex } from '@chakra-ui/react'
+import { foxAssetId } from '@shapeshiftoss/caip'
+import { useState } from 'react'
+
+import { TradeQuote } from './TradeQuote'
+
+type TradeQuotesProps = {
+  isOpen?: boolean
+  isLoading?: boolean
+}
+
+export const TradeQuotes: React.FC<TradeQuotesProps> = ({ isOpen, isLoading }) => {
+  const [activeQuote, setActiveQuote] = useState('THORchain')
```

Comment:
This needs some love, obviously as it isn't wired up yet - but I'll do it in a follow-up PR 👌 

Fine for now as it's behind a feature flag and is just a template.

Reply:
Yep didn't know how we wanted to pass around the selected quote, so just did so dumb logic to show active state.